docs/source/deployment.rst

```diff
@@ -53,3 +53,9 @@ Config apache
 -------------
 
 replace ``PDC_HOSTNAME`` with server's hostname in ``/etc/httpd/conf.d/pdc.conf``
+
+
+Enabled proxy
+-------------
+
+Add ``USE_X_FORWARDED_HOST = True`` in `local_setting.py` file.
```

Comment:
Also a link to Django documentation might be useful: https://docs.djangoproject.com/en/1.9/ref/settings/#use-x-forwarded-host